packages/parchment/lib/src/codecs/markdown.dart

```diff
@@ -1,23 +1,25 @@
 import 'dart:convert';
 
-import 'package:parchment_delta/parchment_delta.dart';
-
-import '../document.dart';
-import '../document/attributes.dart';
-import '../document/block.dart';
-import '../document/leaf.dart';
-import '../document/line.dart';
+import 'package:parchment/parchment.dart';
```

Comment:
When refering to files within the package, we'd rather use relative file imports as per
https://dart.dev/effective-dart/usage#prefer-relative-import-paths